plugins/config/lib/configService.js

```diff
@@ -2046,7 +2147,13 @@ function addJSONFilesToJSON(startingPath,json) {
     fileNames.forEach(function (filename) {
```

Comment:
function should not be called addJSONFilestoJSON anymore, but just "addResource"... because binary is now an option.